components/topbar/user/src/index.scss

```diff
@@ -147,6 +161,21 @@
             $background-color: $bgc-topbar-user-cta,
             $icon-fill: $c-white
           );
+          position: relative;
+        }
+
+        &Notifications {
+          background-color: $c-primary;
```

Comment:
I think we can reuse some common code and create a mixin with it, then apply specific styles in each selector. Can help if you need.